signer/threshold_validator_test.go

```diff
@@ -142,8 +126,21 @@ func TestThresholdValidator2of2(t *testing.T) {
 
 	err = validator.SignProposal("chain-id", &proposal)
 	require.NoError(t, err)
-
 	require.True(t, privateKey.PubKey().VerifySignature(signBytes, proposal.Signature))
+
+	// TODO: Clean up a bit into nice loggnig and asserts/requires.
+	cosigner1JSON, err := json.MarshalIndent(cosigner1, "", "  ")
+	if err != nil {
+		log.Println("cosigner1JSON error:", err.Error())
+	}
+	fmt.Printf("Cosigner1 output \n %s\n", string(cosigner1JSON))
+
+	cosigner2JSON, err := json.MarshalIndent(cosigner2, "", "  ")
+	if err != nil {
+		log.Println("cosigner2JSON error:", err.Error())
+	}
+	fmt.Printf("Cosigner2 output \n %s\n", string(cosigner2JSON))
```

Comment:
Are you going to clean up this TODO?

`fmt.Printf` should almost never be used in test. Prefer `t.Logf`.

Reply:
Fixed!